runtime-sdk/src/modules/core/mod.rs

```diff
@@ -42,6 +52,64 @@ pub enum Error {
     #[error("missing message handler")]
     #[sdk_error(code = 9)]
     MessageHandlerMissing(u32),
+
+    #[error("invalid argument")]
+    #[sdk_error(code = 10)]
+    InvalidArgument,
+
+    #[error("gas overflow")]
+    #[sdk_error(code = 11)]
+    GasOverflow,
+
+    #[error("out of gas")]
+    #[sdk_error(code = 12)]
+    OutOfGas,
+
+    #[error("batch gas overflow")]
+    #[sdk_error(code = 13)]
+    BatchGasOverflow,
+
+    #[error("batch out of gas")]
+    #[sdk_error(code = 14)]
+    BatchOutOfGas,
+}
+
+/// Parameters for the core module.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct Parameters {
+    #[serde(rename = "batch_gas")]
+    pub max_batch_gas: u64,
+}
+
+impl Default for Parameters {
```

Comment:
This was resolved but it still has the custom Default implementation? Also below.

Reply:
woa let me check

Reply:
oh oops I must have gotten confused with similar changes in the account module. of course other PRs would not have touched the new code in core